torch/nn/modules/rnn.py

```diff
@@ -25,6 +26,16 @@ def __init__(self, mode, input_size, hidden_size,
         self.bidirectional = bidirectional
         num_directions = 2 if bidirectional else 1
 
+        if not isinstance(dropout, numbers.Number) or not 0 <= dropout <= 1:
+            raise ValueError("dropout should be a number in range [0, 1] "
+                             "representing the probablity of an element being "
+                             "zeroed")
+        if dropout > 0 and num_layers == 1:
+            raise ValueError("dropout option adds dropout after all but last "
```

Comment:
I feel like it's ok to make this one a warning (e.g. for debugging?). wdyt?

Reply:
Sure.